inventory/src/components/ItemList/BlueprintItemTable.js

```diff
@@ -0,0 +1,263 @@
+          <Table>
+            <TableHead>
+              <TableRow>
+                <TableCell>Item Name</TableCell>
+                <TableCell align={'center'}>Unit</TableCell>
+                <TableCell align={'center'}>Total Qty</TableCell>
+                <TableCell align={'center'}>
+                  Qty To Add (Capped Between 0 & Total Qty)
+                </TableCell>
+                <TableCell></TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
+              {itemsToDisplay.slice(startIndex, endIndex).map((item) => (
+                <TableRow key={item.id} data-testid={`item-row-${item.id}`}>
+                  <TableCell>{item.name}</TableCell>
+                  <TableCell align={'center'}>{item.unit}</TableCell>
+                  <TableCell align={'center'}>{item.quantity}</TableCell>
+                  <TableCell>
+                    <Box display={'flex'} justifyContent={'center'}>
+                      <Button
+                        style={{ border: '0.5px solid' }}
+                        disabled={
+                          item.selectedQty <= 0 || item.selectedQty === ''
+                        }
+                        onClick={() => handleMinusButtonClick(item.id)}
+                      >
+                        <RemoveIcon />
+                      </Button>
+                      <TextField
+                        id='outlined-basic'
+                        label='Qty'
+                        type='number'
+                        sx={{ width: '30%' }}
+                        value={item.selectedQty}
+                        onChange={(e) => {
+                          handleInputChange(
+                            item.id,
+                            e.target.value,
+                            item.quantity,
+                          );
+                        }}
+                        onBlur={(e) => {
+                          handleOnBlur(item.id);
+                        }}
+                        style={{
+                          marginLeft: '5px',
+                          marginRight: '5px',
+                        }}
+                      />
+                      <Button
+                        style={{ border: '0.5px solid' }}
+                        disabled={
+                          item.selectedQty >= item.quantity ||
+                          item.selectedQty === ''
+                        }
+                        onClick={() => handleAddButtonClick(item.id)}
+                      >
+                        <AddIcon />
+                      </Button>
+                    </Box>
+                  </TableCell>
+                  <TableCell>
+                    {item.selectedQty === 0 || item.selectedQty === '' ? (
+                      <RadioButtonUncheckedIcon />
+                    ) : (
+                      <RadioButtonCheckedIcon />
+                    )}
+                  </TableCell>
```

Comment:
This button field does not seem to be mobile responsive, you can try to view it and mobile and make some changes to the style.
Alternatively, I created a NumberInput component in https://github.com/sjb-deck/deck/pull/176/files#diff-f039e6b83abf1e7936c657b1a11ca5ec68dc092e06208237fe6cf1a70ba30405, but you need to incorporate the logic into that component. The component has also yet to be merged